namespace-to-cjs.rb

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env ruby
```

Comment:
This script might be useful to add somewhere else since it will be beneficial for other repos as well.